templates/core/features/ota-updates.html

```diff
@@ -1,218 +1,201 @@
-    <div class="col-7">
-      <p class="p-heading--4">Compressed and read-only</p>
-      <p>The content that ships inside snaps is compressed and remains compressed and read-only through the whole life time of the snap, even during normal use after installation. This means the original package digest and signature is never touched.</p>
-    </div>
-  </div>
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
-  <div class="u-fixed-width">
-    <hr class="p-separator"/>
-  </div>
+{% block content %}
 
-  <div class="row u-equal-height u-vertically-center">
-    <div class="col-7">
-      <p class="p-heading--4">Safe operating system updates</p>
-      <p class="p-heading--5">Resilience and flexibility all the way down.</p> 
-      <p>The kernel and base operating system are handled as snaps as well, so all the benefits that apply to applications also apply to the system foundation. This means not only benefitting from fast and consistent updates to the core, but also graceful error handling with automatic rollbacks on improperly updated kernels.</p>
-    </div>
-    <div class="col-5 u-hide--medium u-hide--small u-align--center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/714e4c31-OTA_Stack_Case.svg",
-        alt="Safe operating system updates",
-        width="300",
-        height="130",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
-
-<section class="p-strip--light">
-  <div class="row u-equal-height">
-    <div class="col-5 u-hide--medium u-hide--small u-align--center">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-            alt="Get in touch",
-            width="281",
-            height="200",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-          }}
-    </div>
-    <div class="col-7">
-      <h2>Get an IoT app store for OTA updates</h2>
-      <p>Get your own infrastructure for secure management and seamless software updates for your fleet of devices.</p>
-      <p>
-        <a href="/core/contact-us?product=core-ota-updates" class="p-button--positive js-invoke-modal">Get in touch</a>
-      </p>
-    </div>
-  </div>
-</section>
+  <section class="p-section--hero">
+    <div class="u-fixed-width p-section--shallow">
+      <h1 class="u-no-margin--bottom">Over-the-air updates for Linux, done right</h1>
+      <h2>Ubuntu Core sets a new standard for Linux device updates, covering  the kernel, the OS and applications.</h2>
+    </div>
+    <div class="row--50-50">
+      <div class="col">
+        <p>
+          Transactional updates for reliability. Deltas minimize network traffic. Digital signatures guarantee integrity and provenance.
+        </p>
+        <p class="p-section--shallow">This is the future of smart things.</p>
+        <hr class="is-muted" />
+        <a class="p-button--positive" href="/internet-of-things/appstore">Get an IoT app store</a>
```

Comment:
These wrap strangely on medium, maybe an inline list with the `u-responsive-realign` class would be better here, but feel free to confirm with design. Difference here:
![image](https://github.com/user-attachments/assets/2763a0b2-05ad-4c21-b654-7c01c7940198)